src/plays/react-todo-app/ReactTodoApp.jsx

```diff
@@ -11,50 +11,81 @@ import './react-todo.css';
 import Input from './components/Input';
 import TodoList from './components/TodoList';
 
+function getLocalStorageTodos() {
```

Comment:
I see an existing hook to handle localstorage operation: https://github.com/reactplay/react-play/blob/main/src/plays/keeper/Hooks/useLocalStorage.js

But this hook is unfortunately inside a play. Could you please move the hook here `https://github.com/reactplay/react-play/tree/main/src/common/hooks` and then use it?

It is better we use reusable code across. Please modify the import of the existing play where it is being used once you move it.